examples/no-repeated-options.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+// How might I throw if an option is repeated?
+
+// 1. const { parseArgs } = require('node:util'); // from node
+// 2. const { parseArgs } = require('@pkgjs/parseargs'); // from package
+const { parseArgs } = require('..'); // in repo
+
+const options = {
+  ding: { type: 'boolean', short: 'd' },
+  beep: { type: 'boolean', short: 'b' }
```

Comment:
By using `multiple: true` and checking that the array value has only 1 item is this an option?

Or do we aim to use `tokens` only?

Reply:
Yes could do using `multiple`. 
But also yes,  this was intended to be an example of using tokens. (And I could say that in an expanded introduction...)